### PR TITLE
 Standardize naming and function parameters in visibilityOptions.js #5669

### LINKED ISSUE #5669: Standardize naming and function parameters in visibilityOptions.js
Currently, the same DOM element is assigned to variable with different names in different methods. 

Additionally, the general design of the methods can be made more consistent by passing in the form containing the various element rather than the element itself. Currently, the caller method has to extract the specific element to be passed into the callee methods, the callee methods than have to use $elem.closest('form') get back the form from which the specific element is extracted. We should just pass in the form containing the element.

### PR DESCRIPTION
**Pull request by YongJieYongJie**
Fixes #5669

### CODE REVIEW

File: src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js [hunk 1]
```diff
@@ -53,38 +91,38 @@ function formatCheckBoxes() {
     // TODO: change class -> name?
     $('input[class*="answerCheckbox"]').change(function() {
         if (!$(this).is(':checked')) {
-            var visibilityOptionsRow = $(this).closest('tr');
-            visibilityOptionsRow.find('input[class*="giverCheckbox"]')
+            var editTabRows = $(this).closest('tr');
```
Comment (JosephineKwa) [inline src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js]: Although the `$` wasn't present before, as we are changing names in this PR, maybe we could also prefix this with a `$` to distinguish jquery objects from regular variables

**Comment by YongJieYongJie** [comment]
@JosephineKwa Ready for review!

File: src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js [hunk 2]
```diff
@@ -179,65 +192,57 @@ function getVisibilityMessage(buttonElem) {
...
+            updateToggleVisibilityPreviewButton($containingForm, false);
+            $containingForm.find('.visibilityOptionsLabel').click();
         }
     });
 }
 
-function updateVisibilityMessageButton($form, isLoadSuccessful) {
-    var visibilityButton = $form.find('.visibilityMessageButton');
+function updateToggleVisibilityPreviewButton($containingForm, isLoadSuccessful) {
+    var visibilityPreviewButton = $containingForm.find('.visibilityMessageButton');
```
Comment (JosephineKwa) [inline src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js]: Do prefix with `$` here too :)

File: src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js [hunk 3]
```diff
@@ -179,65 +192,57 @@ function getVisibilityMessage(buttonElem) {
...
     
-    var radioInput = visibilityButton.find('input[type="radio"]');
+    var radioInput = visibilityPreviewButton.find('input[type="radio"]');
```
Comment (JosephineKwa) [inline src/main/webapp/js/instructorFeedbackEdit/visibilityOptions.js]: Here too

**Comment by YongJieYongJie** [comment]
@JosephineKwa Ready for review! (sorry for not scanning the code thoroughly enough and missing some variables)

**Comment by JosephineKwa** [comment]
Looks good!

File: src/test/resources/pages/instructorFeedbackQuestionEditToTeamToTeam.html [hunk 4]
```diff
@@ -974,42 +974,38 @@ <h4 class="modal-title">
...
-                    <td>
-                      <input class="visibilityCheckbox giverCheckbox1" type="checkbox" value="RECEIVER_TEAM_MEMBERS">
-                    </td>
-                    <td>
-                      <input class="visibilityCheckbox recipientCheckbox1" type="checkbox" value="RECEIVER_TEAM_MEMBERS">
-                    </td>
-                  </tr>
-                </hide>
+                <tr>
+                  <td class="text-left">
```
Comment (wkurniawan07) [inline src/test/resources/pages/instructorFeedbackQuestionEditToTeamToTeam.html]: Yup, this should have been caught beforehand. @YongJieYongJie @JosephineKwa next time be careful, okay?

**Pull request merged**